OrbitGl/GraphTrack.h

```diff
@@ -14,17 +14,21 @@ class TimeGraph;
 
 class GraphTrack : public Track {
  public:
-  explicit GraphTrack(TimeGraph* time_graph);
+  explicit GraphTrack(TimeGraph* time_graph, uint64_t graph_id);
```

Comment:
Remove `explicit` now?

Reply:
I'm all for it if it's on purpose, I was pointing it out as it's something we don't usually do anywhere else.